--- PagePower.qml	2018-09-22 23:20:17.000000000 +0200
+++ PagePower.qml_	2018-10-10 16:04:56.449072074 +0200
@@ -33,17 +33,9 @@
             case 2:
                 sddm.suspend();
                 break;
-
-            case 3:
-                sddm.hibernate();
-                break;
-
-            case 4:
-                sddm.hybridSleep();
-                break;
         }
     }
-    
+
     ColumnLayout
     {
         id: powerListContainer
@@ -103,41 +95,6 @@
             onEntered: selectedIndex = 2
             onClicked: { selectedIndex = 2; execute() }
         }
-
-        LoopListPowerItem
-        {
-            id: powerHibernateButton
-            title: qsTr("Hibernate")
-            distance: selectedIndex == 3 ? 1.0 : 0.6
-            hover: selectedIndex == 3
-            icon: "icons/hibernate.svg"
-
-            Layout.alignment: Qt.AlignVCenter
-            Layout.minimumHeight: 48
-
-            visible: debug.canHibernate
-
-            onEntered: selectedIndex = 3
-            onClicked: { selectedIndex = 3; execute() }
-        }
-
-        LoopListPowerItem
-        {
-            id: powerHybridSleepButton
-            title: qsTr("Hybrid Sleep")
-            distance: selectedIndex == 4 ? 1.0 : 0.6
-            hover: selectedIndex == 4
-            icon: "icons/hybrid-sleep.svg"
-
-            //Layout.fillHeight: true
-            Layout.alignment: Qt.AlignVCenter
-            Layout.minimumHeight: 48
-
-            visible: debug.canHybridSleep
-
-            onEntered: selectedIndex = 4
-            onClicked: { selectedIndex = 4; execute() }
-        }
     }
 
     function scroll_up()
@@ -158,7 +115,7 @@
 
         while (!actionFound)
         {
-            if (result >= 4)
+            if (result >= 2)
                 result = 0
             else
                 result++
@@ -178,14 +135,6 @@
                 case 2:
                     if (debug.canSuspend) actionFound = true
                     break
-
-                case 3:
-                    if (debug.canHibernate) actionFound = true
-                    break
-
-                case 4:
-                    if (debug.canHybridSleep) actionFound = true
-                    break
             }
 
         }
@@ -202,7 +151,7 @@
         while (!actionFound)
         {
             if (result <= 0)
-                result = 4
+                result = 2
             else
                 result--
 
@@ -221,14 +170,6 @@
                 case 2:
                     if (debug.canSuspend) actionFound = true
                     break
-
-                case 3:
-                    if (debug.canHibernate) actionFound = true
-                    break
-
-                case 4:
-                    if (debug.canHybridSleep) actionFound = true
-                    break
             }
 
         }
@@ -240,4 +181,4 @@
     Keys.onDownPressed: scroll_up()
     Keys.onEnterPressed: execute()
     Keys.onReturnPressed: execute()
-}
\ No newline at end of file
+}
